lib/JSON/Validator/OpenAPI.pm

```diff
@@ -10,6 +11,28 @@ my %COLLECTION_RE = (pipes => qr{\|}, csv => qr{,}, ssv => qr{\s}, tsv => qr{\t}
 
 has _json_validator => sub { state $v = JSON::Validator->new; };
 
+{    # proxy methods for compatibility
+  my @proxy_methods = qw<
```

Comment:
I actually like `qw<>` because `<>` does the same in Perl6 but for consistency's sake I tried to keep close to your style. You're right.